docs/developer/migrate/from-ethereum.md

```diff
@@ -97,7 +96,7 @@ You can [view the implementation here.](https://explorer.celo.org/address/0xaa93
     Celo transaction objects include the same fields as Ethereum transaction objects, plus the feeCurrency field.
```

Comment:
I'm not sure this is up to date. The latest references I'm aware of are https://docs.celo.org/what-is-celo/using-celo/protocol/transaction/transaction-types and  https://specs.celo.org/tx_types.html, but I don't find them crystal clear in terms of what's good to highlight here